src/util/apis/generated/models/aggregatedBillingInformation.js

```diff
@@ -1,175 +1,133 @@
+   * @property {string} [timestamp] The ISO 8601 datetime of last modification
+   * @property {string} [id] ID of the user or organization
+   * @property {object} [billingPlans]
+   * @property {object} [billingPlans.buildService]
+   * @property {boolean} [billingPlans.buildService.canSelectTrialPlan] Can
    * customer select trial plan for that service (if it exists)?
-   * @member {string} [billingPlans.buildService.lastTrialPlanExpirationTime]
+   * @property {string} [billingPlans.buildService.lastTrialPlanExpirationTime]
    * Expiration time of the last selected trial plan. Will be null if trial
    * plan was not used.
-   * @member {object} [billingPlans.buildService.currentBillingPeriod]
-   * @member {string}
+   * @property {object} [billingPlans.buildService.currentBillingPeriod]
+   * @property {string}
    * [billingPlans.buildService.currentBillingPeriod.startTime] Inclusive start
    * of the period
-   * @member {string} [billingPlans.buildService.currentBillingPeriod.endTime]
-   * Exclusive end of the period.
-   * @member {object}
+   * @property {string}
+   * [billingPlans.buildService.currentBillingPeriod.endTime] Exclusive end of
+   * the period.
+   * @property {object}
    * [billingPlans.buildService.currentBillingPeriod.byAccount]
-   * @member {number}
+   * @property {number}
    * [billingPlans.buildService.currentBillingPeriod.byAccount.count] Number of
    * instances of the billing plan.
-   * @member {object}
+   * @property {object}
    * [billingPlans.buildService.currentBillingPeriod.byAccount.plan]
-   * @member {string}
+   * @property {string}
    * [billingPlans.buildService.currentBillingPeriod.byAccount.plan.id] The
    * Billing Plan ID
-   * @member {string}
+   * @property {string}
    * [billingPlans.buildService.currentBillingPeriod.byAccount.plan.version]
    * Version of the Billing Plan schema
-   * @member {number}
+   * @property {number}
    * [billingPlans.buildService.currentBillingPeriod.byAccount.plan.price]
    * Price of the Billing Plan
-   * @member {string}
+   * @property {string}
    * [billingPlans.buildService.currentBillingPeriod.byAccount.plan.paymentSource]
    * Service that receives payments for this billing plan. Possible values
    * include: 'None', 'AppCenter', 'GitHub', 'Xtc'
-   * @member {string}
+   * @property {string}
    * [billingPlans.buildService.currentBillingPeriod.byAccount.plan.service]
    * Name of the service that the plan applies to. Possible values include:
-   * 'Build', 'Push', 'Test'
-   * @member {object}
+   * 'Build', 'Test'
+   * @property {object}
    * [billingPlans.buildService.currentBillingPeriod.byAccount.plan.limits]
-   * @member {object}
+   * @property {object}
    * [billingPlans.buildService.currentBillingPeriod.byAccount.plan.attributes]
-   * @member {string}
+   * @property {string}
    * [billingPlans.buildService.currentBillingPeriod.byAccount.plan.parentId]
-   * @member {object} [billingPlans.pushService]
```

Comment:
I'd like to reach out to the Push service and check that the changes to the billing API around Push is intentional before we release this version of the CLI. If you know the best person to reach out to - please let me know.